src/jit/gentree.cpp

```diff
@@ -1329,6 +1329,62 @@ GenTreeCall::GetOtherRegMask() const
     return resultMask;
 }
 
+//-----------------------------------------------------------------------------------------
+// TreatAsHasMultiRegRetVal: 
```

Comment:
I'm not sure what you mean by "align the comment to the name of function"?
Can you cut and paste exactly what you want to see here?